### ISSUE TITLE
Saving to Request.prof_file raises DataError for paths longer than 59 characters

### ISSUE THREAD

**Issue by smcoll** [jazzband/silk#201]
The decorator is raising a DataError when it tires to save a `.prof` for any path greater than 59 characters, since the `Request.prof_file` is 100 characters by default, and the filename itself is 41 characters.

```
django.db.utils.DataError: value too long for type character varying(100)
```

In my case, the project path for local development is more than 60 characters, which means the file is saving to disk, but no profile data is being saved to the database.

**Comment by avelis** [comment]
@smcoll If my memory serves me correct I think you can specify a path to store the `.prof` files. 

One suggestion is to create an symbolic link in your environment to get around the 100 character limit.

**Comment by smcoll** [comment]
@avelis that will do for now, but it'd be reasonable to increase the column length, right?  i'm making a PR now.

**Comment by srinivasreddy** [comment]
this issue can be closed since #203 is merged

**Comment by smcoll** [comment]
@srinivasreddy no, #203 increased the limit by 200 characters, but the underlying issue is still the same.  It needs to be determined how silk will handle a case where a binary cannot be saved.
